nexteuropa_newsroom.admin.inc

```diff
@@ -221,7 +221,7 @@ function nexteuropa_newsroom_admin_settings($form, &$form_state) {
   $form['newsroom_universe']['nexteuropa_newsroom_allowed_ips'] = [
     '#type' => 'textfield',
     '#title' => t('IP addresses allowed for import'),
-    '#default_value' => variable_get('nexteuropa_newsroom_allowed_ips', '127.0.0.1'),
+    '#default_value' => variable_get('nexteuropa_newsroom_allowed_ips', '127.0.0.1,158.167.165.192,158.167.165.191,158.167.209.222,158.167.240.147,158.167.209.223,158.167.210.159,158.167.210.158'),
```

Comment:
Please put these IP's into a variable array with documentation on what IP is what.
Then implode the variable for this default value.